### PR TITLE
DM-29502: Upgraded black to version 20.8b1 and removed the use of asy…

### PR DESCRIPTION
**Pull request by wvreeven**
…nctest.

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -20,11 +20,12 @@ build:
 
 test:
   requires:
-    - ts-conda-build
+    - ts-conda-build =0.3
     - ts-simactuators
     - ts-dds
     - ts-idl {{ idl_version }}
     - ts-salobj {{ salobj_version }}
+    - black
```
Comment (couger01) [inline conda/meta.yaml:28]: Do you need black here since it is provided by the metapackage?

**Review by couger01 — CHANGES_REQUESTED** [review]
One question otherwise looks good.

File: conda/meta.yaml (hunk 1 above)
Comment (wvreeven) [reply conda/meta.yaml:28]: Nope so I removed it.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**